tdameritrade/auth/__init__.py

```diff
@@ -1,11 +1,12 @@
 import os
 import os.path
 import sys
-import requests
 import time
-from selenium import webdriver
-from shutil import which
 import urllib.parse as up
+from shutil import which
+
+import requests
+from selenium import webdriver
```

Comment:
move selenium import into the function call so that its not strictly required

Reply:
I'm confused about how these requests are working. Are you implementing changes in this or asking me to do them?